tox.ini

```diff
@@ -1,17 +1,35 @@
 [tox]
-envlist = black,type,py36,py37,py38,py39
+minversion = 1.4.2
+envlist = linters
+skipsdist = True
 skip_missing_interpreters = true
 
 [testenv]
-deps =
-    -r{toxinidir}/requirements-test.txt
-    -c{toxinidir}/tests/constraints.txt
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
 commands =
     py.test -v
 
 [testenv:black]
+install_command = pip install {opts} {packages}
 commands =
-    black --check .
+  black -v {toxinidir}
+
+[testenv:linters]
+install_command = pip install {opts} {packages}
+commands =
+  black -v --diff --check {toxinidir}
+  flake8 {posargs}
+  yamllint -s .
+
+[flake8]
+# E123, E125, E203 skipped as they are invalid PEP-8.
+
+show-source = True
+ignore = E123,E125,E203,W503
+max-line-length = 160
+builtins = _
+exclude = .git,.tox,*__init__.py
```

Comment:
what is the reason for ignoring *__init__.py

Reply:
> what is the reason for ignoring ***init**.py

Flake8 throws  F401 error, because the imported module is not used in the same file. This happens for most of the __init__ files. These files have only import statements. So I decided to exclude them from flake8 check.

Reply:
I would ignore f401 for now, rather then skipping the whole file. Then we can work to bring it into flake8 in follow up patches